file/pkg/service/service.go

```diff
@@ -251,6 +234,7 @@ func (f *fileService) CreateFileShare(ctx context.Context, in *pb.CreateFileShar
 	log.Debugf("Create File share response, fileshare: %+v\n", out.Fileshare)
 
 	log.Info("Created file share successfully.")
+	go f.SyncFileShare(ctx, out.Fileshare, backend.Backend)
```

Comment:
Please put a comment for why this is required

Reply:
Done